terraform/README.md

```diff
@@ -0,0 +1,64 @@
+## migraiton of current static ip
```

Comment:
migration

Reply:
I think after the manual migration we should remove this file anyway?

Reply:
yes this will be removed after migration